spec/controllers/queue_items_controller_spec.rb

```diff
@@ -155,45 +154,278 @@
       end
     end
   end
+
+  describe "POST update" do
+    
+    context "when a user is signed in" do
+      before { session[:current_user_id] = pete.id }
+      
+      context "when position changes pass validations" do
+
+        context "when the user does not own one of the queue items." do
+          let(:qi_1) { Fabricate(:queue_item, position: 1) }
+
+          before { post :update, queue_items: { "#{qi_1.id}" => { position: 2, user_rating: 2 } } }                                      
+
```

Comment:
I would make your `before` multiline with `do`/`end` because you have so many curly braces on this line, but that is definitely a personal preference.